TWLight/static/css/local.css

```diff
@@ -774,3 +774,8 @@ CLEARING COLUMNS
   color: #fff;
 }
 
+.homeBtn:hover {
+  background-color:#337ab7;
+  border-color:#337ab7;
+  color: #fff;
+}
```

Comment:
I'm not sure about this change here. I believe we should keep the button's default CSS. @Samwalton9 @jsnshrmn what do you think?

This is the current behavior of the default buttons on hover:
![Screen Shot 2020-10-06 at 11 52 36](https://user-images.githubusercontent.com/7854953/95232686-79039c00-07ca-11eb-9a2f-cb78ee9b49ab.png)

This is the changed behavior of the default buttons on hover:
![Screen Shot 2020-10-06 at 11 52 46](https://user-images.githubusercontent.com/7854953/95232775-90428980-07ca-11eb-8b6d-6fcafd2600b6.png)

If we keep the changes you made, I think the CSS class should be renamed to something different, like `default-btn-override:hover`

Reply:
It's not clear to me what problem the css change is solving or how it relates to the pr. I'd exclude the change.

Reply:
Should I rename the CSS class right now???

Reply:
Actually It's my mistake that I had put this CSS change in this PR. Instead I should file a another PR for this CSS change.
If you don't like this change, I can remove this...

Reply:
Let's back up a step: why did you change the css?

Reply:
Is there a phab issue for the problem that the css change is trying to solve?

Reply:
I thought that this change would look nice and more attractive than the default CSS

Reply:
No, but I had discuss this change with Sam on email

Reply:
Ah, yeah, I can't really evaluate in that case, but it doesn't seem like an obvious improvement to me since it's a common bootstrap UI metaphor to simply darken the existing color when hovering over an interactive UI element instead of changing colors completely. Will wait to hear back from @Samwalton9 either for evaluation or for more information. I agree with @suecarmol that if we want to accept the change, the class name shouldn't be "homeBtn".

Reply:
Agreed that the default behaviour is fine, I don't see a need to add complexity for such a minor change.